server/registry/internal/storage/client.go

```diff
@@ -70,6 +70,7 @@ func NewClient(ctx context.Context, driver, dsn string) (*Client, error) {
 			c.close()
 			return nil, grpcErrorForDBError(ctx, err)
 		}
+		db.Exec("PRAGMA foreign_keys = ON")
```

Comment:
Does this affect paging?

Reply:
No, that shouldn't be there... my git history is screwy. Will remove.